assets/js/payment-method-extensions/payment-methods/stripe/credit-card/payment-method.js

```diff
@@ -76,14 +76,16 @@ const CreditCardComponent = ( { billing, eventRegistration, components } ) => {
 				/>
 			) }
 			<div className="wc-blocks-credit-card-images">
-				{ getStripeServerData().icons.map( ( { id, url, alt } ) => (
-					<img
-						src={ url }
-						alt={ alt }
-						key={ id }
-						className={ `.wc-blocks-credit-${ id }-icon wc-blocks-credit-cart-icon` }
-					/>
-				) ) }
+				{ Object.entries( getStripeServerData().icons ).map(
```

Comment:
wasn't there a problem with using `Object.entries` with IE11?  If so you can probably just map through the keys and then reference each value using the provided keys:

```js
const icons = getStripeServerData().icons;
Object.keys( icons ).map( ( id ) => {
    const { url, alt } = icons[ id ];
    // ...
} );
```

Reply:
It was [`Object.fromEntries`](https://caniuse.com/#feat=mdn-javascript_builtins_object_fromentries) that caused issues (not being supported in edge as well), granted after a google search it seems IE11 doesn’t support [`Object.entries`](https://caniuse.com/#feat=object-entries) nor does it support [`Object.values`](https://caniuse.com/#feat=object-values) as well, to my full surprise, good catch, will change it.

Reply:
Ahh, well I _know_ we have polyfills for Object.values (and we likely do for `Object.entries`) too in which case this will be okay.